### PR TITLE
Fix typos and make minor improvements

### PR DESCRIPTION
**Pull request by bilalshareef**
resolve nearform/hub-draft-issues#346

### CODE REVIEW

File: slides.md [hunk 1]
```diff
@@ -697,9 +697,9 @@ You can reference the [`--import` official documentation](https://nodejs.org/api
 
 # A12 The problem
 
-- Run in the terminal `node --test --experimental-test-coverage`.
+- Run in the terminal `node --test --experimental-test-coverage ./test/index.test.js`.
```
Comment (simoneb) [inline slides.md:700]: are you sue that this is needed? the test runner should be able to find the tests automatically

File: src/a03-parallel-testing/test/multiply.test.js [hunk 2]
```diff
@@ -2,19 +2,19 @@ import assert from 'node:assert'
 import { test } from 'node:test'
 import { heavyComputationMultiply } from '../src/index.js'
 
-test('test some heavy computation sum functions', async () => {
+test('test some heavy computation multiply functions', async () => {
   await test('2 * 100_000_000', async () => {
-    const sum = await heavyComputationMultiply(2, 100_000_000)
-    assert.deepStrictEqual(sum, 200_000_000)
+    const multiply = await heavyComputationMultiply(2, 100_000_000)
```
Comment (simoneb) [inline src/a03-parallel-testing/test/multiply.test.js:7]: this should be called product

File: slides.md (hunk 1 above)
Comment (bilalshareef) [reply slides.md:700]: This change is in relation with the next line which reads "Watch the coverage not being 100%.".
When we run just `node --test --experimental-test-coverage`, it picks `solution.test.js` file as well and shows coverage as 100%.
So, adding `./test/index.test.js` makes sure coverage reported is not 100%.

Comment (simoneb) [reply slides.md:700]: ah ok, makes sense then 👌

File: src/a03-parallel-testing/test/multiply.test.js (hunk 2 above)
Comment (bilalshareef) [reply src/a03-parallel-testing/test/multiply.test.js:7]: Yeah, I agree. Will make the change.

Comment (bilalshareef) [reply src/a03-parallel-testing/test/multiply.test.js]: Done

**Review by simoneb — APPROVED** [review]

**Comment by simoneb** [comment]
I thought you were going to remove the custom install instructions from step 11.

**Review by simoneb — APPROVED** [review]

**Pull request merged**